yew-ssr-actix-web/README.md

```diff
@@ -0,0 +1,8 @@
+# Yew Server-side Rendering with actix-web Example
+
+This example demonstrates yew server-side rendering with actix-web.
+
+This demo need the **development version** of yew.
```

Comment:
why is the dev version required ?

Reply:
SSR would be supported in the Yew Next version, not the current latest version(0.19.3).

Reply:
Ahh that's exciting, we'll wait for its release.